index.html

```diff
@@ -703,5 +706,32 @@ <h2 class="banner-sub">LET’S CONNECT</h2>
         />
       </svg>
     </section>
+
+    <!-- Contact form -->
+    <section class="contact-form">
+      <h3 class="contact-header">Contact me</h3>
+      <p class="contact-desc">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+
+      <form
+        id="form-ux"
+        class="form-info"
+        action="https://formspree.io/f/form-ux"
+        method="post"
+      >
+        <input type="text" placeholder="Lorem Ipsum" required maxlength="50" />
+        <input type="email" placeholder="loremipsum@gmail.com" required />
+        <textarea
+          placeholder="Write your message here ..."
+          maxlength="500"
+          required
+        ></textarea>
+
+        <button class="show form-btn">Get in touch</button>
+      </form>
```

Comment:
- [ ] Kindly, your form is not successfully integrated with [formspree service](https://formspree.io/html). Please check well your ```form id```and make sure that it is being submitted successfully 🙏 

### Currently what I get after submitting 👇 
![image](https://user-images.githubusercontent.com/102757126/228255962-e1b51c40-181c-49f0-b241-3b3652b1607c.png)

- [ ] Please as recommended in the project requirements, our name input should accept up to **30 characters**, Kindly change the ```maxlength=50``` to ```maxlength=30``` as suggested in the project requirements 😁😁